cpu/sam0_common/periph/rtc_rtt.c

```diff
@@ -267,6 +278,93 @@ void rtt_init(void)
     NVIC_EnableIRQ(RTC_IRQn);
 }
 
+#if RTC_NUM_OF_TAMPERS
+
+static rtc_state_t tamper_cb;
+
+/* check if pin is a RTC tamper pin */
+static int _rtc_pin(gpio_t pin)
+{
+    for (unsigned i = 0; i < ARRAY_SIZE(rtc_tamper_pins); ++i) {
+        if (rtc_tamper_pins[i] == pin) {
+            return i;
+        }
+    }
+
+    return -1;
+}
+
+void rtc_tamper_init(void)
+{
+    if (IS_ACTIVE(MODULE_PERIPH_RTC) ||
+        IS_ACTIVE(MODULE_PERIPH_RTT) ||
+        _power_is_on()) {
+        return;
+    }
+
+    _rtt_clock_setup();
+    _poweron();
+
+    /* disable all interrupt sources */
+    RTC->MODE0.INTENCLR.reg = RTC_MODE0_INTENCLR_MASK;
+
+    NVIC_EnableIRQ(RTC_IRQn);
+}
```

Comment:
Why not have this in auto-init as for most modules?

Reply:
Good idea!